diff --git a/node_modules/@actual-app/api/connection.js b/node_modules/@actual-app/api/connection.js
index 76d2544..d40940c 100644
--- a/node_modules/@actual-app/api/connection.js
+++ b/node_modules/@actual-app/api/connection.js
@@ -41,6 +41,7 @@ async function init(socketName) {
     // TODO: This could spawn Actual automatically. The ideal solution
     // would be to bundle the entire backend and embed it directly
     // into the distributed library.
+    initialized = false;
     throw new Error("Couldn't connect to Actual. Please run the app first.");
   }
 
